src/views/product-integration/components/component-tab-content/index.tsx

```diff
@@ -4,21 +4,28 @@ import {
 	Variable as ApiVariable,
 	VariableGroup,
 } from 'lib/integrations-api-client/release'
-import ReactMarkdown from 'react-markdown'
+import { MDXRemote, type MDXRemoteSerializeResult } from 'next-mdx-remote'
+import defaultMdxComponents from 'layouts/sidebar-sidecar/utils/_local_platform-docs-mdx'
 import SearchableVariableGroupList from '../searchable-variable-group-list'
 import { Variable } from '../variable-group-list'
 import s from './style.module.css'
 
 interface ComponentTabContentProps {
-	component: ReleaseComponent
+	component: ReleaseComponent & { readmeMdxSource: MDXRemoteSerializeResult }
 }
 
 export default function ComponentTabContent({
 	component,
 }: ComponentTabContentProps) {
+	const readmeContent = (
+		<MDXRemote
+			{...component.readmeMdxSource}
+			components={defaultMdxComponents({})}
+		/>
+	)
 	return (
 		<div className={s.componentTabContent}>
-			{component.readme && <ReactMarkdown>{component.readme}</ReactMarkdown>}
+			{component.readmeMdxSource ? readmeContent : null}
```

Comment:
There appears to be a margin-top here (I believe coming from the H2).

Is there something we can do about this (e.g. first child no margin top?)

<img width="582" alt="CleanShot 2022-12-19 at 10 12 48@2x" src="https://user-images.githubusercontent.com/2105067/208492310-80d4ea8a-4cb3-471c-a992-bcb2c923aeef.png">

Reply:
I'll see what I can do!

(note, it's being defined here https://github.com/hashicorp/dev-portal/blob/kw/improve-markdown-rendering/src/components/dev-dot-content/mdx-components/mdx-headings/mdx-headings.module.css#L44-L48, and applied when we opt into using `MdxH1`, `MdxH2`, etc for our markdown heading elements)